### PR TITLE
Remove gossip

### PR DESCRIPTION
**Pull request by povilasv**
## Changes

Remove gossip :fire:

## Issue

https://github.com/improbable-eng/thanos/issues/734

## Verification

Tests pass

Tested this in my k8s env, couldn't find a issue

### CODE REVIEW

**Comment by povilasv** [comment]
I plan to test this in dev on monday,

File: test/e2e/spinup_test.go [hunk 1]
```diff
@@ -154,9 +153,7 @@ func receiver(i int, config string) cmdScheduleFunc {
 
 func querier(i int, replicaLabel string, staticStores ...string) cmdScheduleFunc {
```
Comment (bwplotka) [inline test/e2e/spinup_test.go]: I think we can remove this one, right?

**Comment by povilasv** [comment]
tested a bunch of components in my dev k8s cluster, it looks like it works :D

**Comment by GiedriusS** [comment]
I guess it's not part of it but we should also remove gossip info from the example Grafana dashboards and the `benchmark/cmd/thanosbench/*`. Care to create an issue for that? It's not a blocker :stuck_out_tongue:

**Comment by povilasv** [comment]
@GiedriusS 

thanks

PTAL

**Comment by povilasv** [comment]
Issue: https://github.com/improbable-eng/thanos/issues/1180

**Comment by brancz** [comment]
Love it! Let's get rid of it finally!

**Comment by povilasv** [comment]
:tada:

**Pull request merged**